composer.json

```diff
@@ -24,7 +24,7 @@
     ],
     "require": {
         "php": "^5.6 || ^7.0",
-        "egeloen/ckeditor-bundle": "^4.0 || ^5.0 || ^6.0",
+        "friendsofsymfony/ckeditor-bundle": "^1.0",
```

Comment:
Maybe move this to suggest?

Reply:
If I am going to check types, shouldn't it be in require dev?